docs/Examples.md

```diff
@@ -65,21 +87,24 @@ If you want more indexers, just update it to include a `replicas` parameter:
 
 ```yaml
 cat <<EOF | kubectl apply -f -
-apiVersion: enterprise.splunk.com/v1alpha2
+apiVersion: enterprise.splunk.com/v1alpha3
```

Comment:
Generic question: wouldn't apiVersions change to beta at some point?

Reply:
At some later point yes. Maybe we need to keep updating docs in such cases.